src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -211,18 +242,6 @@ public int MaxModelValidationErrors
         public bool RequireHttpsPermanent { get; set; }
 
 
-        /// <summary>
-        /// Gets or sets the option to determine if model binding should convert all exceptions (including ones not related to bad input)
-        /// that occur during deserialization in <see cref="IInputFormatter"/>s into model state errors.
-        /// This option applies only to custom <see cref="IInputFormatter"/>s.
-        /// Default is <see cref="InputFormatterExceptionModelStatePolicy.AllExceptions"/>.
-        /// </summary>
-        public InputFormatterExceptionModelStatePolicy InputFormatterExceptionModelStatePolicy
```

Comment:
I'm alphabetizing these things as I go. There's a ton of them so it seems worth doing. I'll another PR at the end to cover the rest.